tests/lib_files_test.py

```diff
@@ -240,3 +235,37 @@ def test_warpvnd_import(fc):
             simulationType=fc.sr_sim_type,
         ),
     )
+
+
+def _get_file(fc, api_name):
+    from pykern import pkunit, pkcompat
+    from sirepo import sim_data
+    from pykern.pkcollections import PKDict
+    from pykern.pkunit import pkre, pkeq
+
+    d = fc.sr_sim_data("Sample from Image")
+    s = sim_data.get_class(fc.sr_sim_type)
+    r = fc.sr_get(
+        api_name,
+        params=PKDict(
+            simulation_type=fc.sr_sim_type,
+            filename="sample.tif",
+        ),
+        data=PKDict(),
+        redirect=False,
+    )
+    pkre("/tif", r.mimetype)
+    f = s.lib_file_resource_path("sample.tif")
+    r = fc.sr_post_form(
+        "uploadFile",
```

Comment:
This should be renamed to uploadLibFile. It does need simulation_id unlike download_file, because it runs checks with the id.

api_deleteFile should be renamed api_deleteLibFile.

Reply:
[20adaad](https://github.com/radiasoft/sirepo/pull/6348/commits/20adaadc1c87c4ecc0f0b07e81f156c1d2297013)

[bb8adce](https://github.com/radiasoft/sirepo/pull/6348/commits/bb8adce741ba12dedf036c5f69996e41804739e9)